src/schematics/deploy/actions.ts

```diff
@@ -159,7 +159,9 @@ export default async function deploy(
   }
 
   try {
-    await firebaseTools.use(firebaseProject, { project: firebaseProject });
+    await firebaseTools.use(firebaseProject, { project: firebaseProject }).then(() => {
+      context.logger.info(`🔥 Your configured Firebase project is '${firebaseProject}'`);
```

Comment:
You don't need .then here,
just log on the next line.

Reply:
Done!